frontend/src/utils/getAccessToken.ts

```diff
@@ -0,0 +1,11 @@
+const accessTokenGetter = () => {
+  const cookies = document.cookie.split(';');
+  for (let cookie = 0; cookie < cookies.length; cookie++) {
+    if (/accessToken=/.test(cookies[cookie])) {
+      const accessToken = cookies[cookie].slice(cookies[cookie].indexOf('=') + 1);
+      return accessToken;
```

Comment:
Maybe it will be better to use some library instead of describing bike from zero?

Reply:
used library js-cookie